webrecorder/web-replay-gen
# Render <replay-web-page> in single static webpage

(https://github.com/webrecorder/web-replay-gen/issues/3) Sets up Lit + ReplayWeb.page lib

### Manual testing
1. Run `npm install` to bring in new dependencies
2. Set up local dev config [per instructions](https://github.com/webrecorder/web-replay-gen/blob/replay-web-page/README.md#development)
3. Run server with `npm run serve`
4. Go to http://localhost:8080 and click archive links. Verify web archive loads
5. Update or omit `replay.version` in config file. Verify correct version loads, or latest loads if omitted
6. Go to `_site/sitemap.xml` Verify sitemap is generated as expected

### TODO
Styling, tracking here: https://github.com/webrecorder/web-replay-gen/issues/6

### Opinions
There is an experimental [eleventy-lit plugin](https://github.com/lit/lit/tree/main/packages/labs/eleventy-plugin-lit#roadmap). I didn't use it since there hasn't been a stable release yet. We may want to migrate to it once stable.
